numba_cuda/numba/cuda/tests/cudapy/test_cooperative_groups.py

```diff
@@ -149,6 +157,32 @@ def test_max_cooperative_grid_blocks(self):
         self.assertEqual(blocks1d, blocks2d)
         self.assertEqual(blocks1d, blocks3d)
 
+    @skip_unless_cc_60
+    def test_external_coorperative_func(self):
+        sig = signature(
+            CPointer(int32),
+        )
+        cta_barrier = cuda.declare_device(
+            "cta_barrier", sig=sig, use_cooperative=True
```

Comment:
The general idea I'm getting at with the comment below (will not have correct formatting or be in the right place to use the definition of `src`, I suspect):

```suggestion
            "cta_barrier", sig=sig, link=[src], use_cooperative=True
```

Reply:
Yup, will address together with the comment below.

Reply:
Done in 2efcc47